qbx_vehiclepush/client.lua

```diff
@@ -1,4 +1,4 @@
----@alias direction nil | 'right' | 'left' | 'front' | 'back'
+---@alias Direction? 'right' | 'left' | 'front' | 'back'
```

Comment:
I meant to add the question mark wherever you're using it, not at the definition. One place where you use it is above `local pushingControl = nil`